packages/frontend/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -203,28 +152,15 @@ export const RootAppSidebar = memo(
           </MenuItem>
         </SidebarContainer>
         <SidebarScrollableContainer>
-          <FavoriteList docCollection={docCollection} />
-          <CategoryDivider label={t['com.affine.rootAppSidebar.collections']()}>
-            <AddCollectionButton node={node} onClick={handleCreateCollection} />
-          </CategoryDivider>
-          <CollectionsList
-            docCollection={docCollection}
-            onCreate={handleCreateCollection}
-          />
+          {runtimeConfig.enableOrganize && <ExplorerOrganize />}
+          <ExplorerFavorites />
```

Comment:
so cool so correct. all other menu items should follow the impl standard. a todo noted